test/verify/check-selinux

```diff
@@ -231,7 +231,11 @@ class TestSelinux(testlib.MachineCase):
             # See https://github.com/SELinuxProject/selinux/commit/3a9b4505b
             # Combining of versions < 3.0 with versions >= 3.0 provides a bit
             # different outputs.
-            return script.replace("permissive -D\n", "")
+            script = script.replace("permissive -D\n", "")
+            # likewise, this only started to exist in Fedora 40
+            if not m.image.startswith('fedora-4'):
```

Comment:
This looks wrong -- you most certainly need it for rhel-9-4 and possibly 9-5 runs as well? Note that this applies to ` self.machines["ansible_machine"]` (the secondary one which always runs TEST_OS_DEFAULT), not `m` (which is TEST_OS, aka. `m.image`)